src/app/services/book.service.ts

```diff
@@ -1,16 +1,29 @@
 import {Injectable} from '@angular/core';
-import {Http, Response} from '@angular/http';
+import {Http, Headers} from '@angular/http';
 import {Observable} from 'rxjs/Observable';
+import {Book} from '../models/book.model';
 
 @Injectable()
 export class BookService {
+  private headers = new Headers({'Content-Type': 'application/json'});
 
-  private getBooksUrl = 'http://milenabooks.azurewebsites.net/api/Books';
+  private getPostBooksUrl = 'http://milenabooks.azurewebsites.net/api/Books';
```

Comment:
It's generally accepted to use the API URL ('http://milenabooks.azurewebsites.net/api/) as a global constant in the project. Especially when you have different environments and you want to build the project with API urls for the relevant environment. 

Plus, if you had more entity types, you can reuse that constant.